PhotoKeeper/PhotoPreviewViewController.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  PhotoPreviewViewController.swift
+//  PhotoKeeper
+//
+//  Created by Devin Argenta on 5/17/18.
+//  Copyright © 2018 Devin Argenta. All rights reserved.
+//
+
+import UIKit
+
+class PhotoPreviewViewController: UIViewController, UITextFieldDelegate, UITextViewDelegate{
+    // MARK: UI Stuff
+    let store = PhotoStore.shared
+    var image = UIImage()
+    let metaForm = UIView()
+    let titleField = UITextField()
+    let descriptionField = UITextView()
+    let submitButton = UIButton()
+    let iView = UIImageView()
+    
+    // MARK: colors
+    let blueblack = UIColor(hue: 0.6333, saturation: 0.15, brightness: 0.64, alpha: 1.0)
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupView()
+        handleKeyboardEvents()
+
+
+    }
+    
```

Comment:
maybe add a `viewDidShow` override to call `becomeFirstResponder`? if you think that's how you want it to work